Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "acl-2.2.52.src.tar.gz" "acl" "host-acl" "add-missing-stdio-header" "\
        include/misc.h \
    ";


===============================================================================
--- a/include/misc.h	2013-04-30 17:07:09.000000000 +0200
+++ b/include/misc.h	2018-01-06 17:11:34.000000000 +0100
@@ -14,6 +14,7 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+#include <stdio.h>
 
 extern int __acl_high_water_alloc(void **buf, size_t *bufsize, size_t newsize);
 
